### PR TITLE
Document "Fix missing seekpoints for FLAC files" option

### PR DESCRIPTION
**Pull request by phw**
### Summary

This is a…

- [ ] Correction
- [x] Addition
- [ ] Restructuring
- [ ] Minor / simple change (like a typo)
- [ ] Other

### Reason for the Change

Document the new "Fix missing seekpoints for FLAC files" option from PICARD-2411

### CODE REVIEW

File: config/options_before_tagging.rst [hunk 1]
```diff
@@ -28,6 +28,11 @@
    Check to remove APEv2 tags from MP3 files – ID3 is recommended for MP3 files. Picard will write ID3 tags to MP3 files
    regardless of this setting.
 
+**Fix missing seekpoints for FLAC files**
+
+   Some software has issues handling FLAC files that have an empty seek table metadata block. This options removes the
```
Comment (rdswift) [inline config/options_before_tagging.rst:33]: Perhaps "This option removes..." rather than "options"?

**Review by rdswift — CHANGES_REQUESTED** [review]
Looks good to me.  Thanks.

File: config/options_before_tagging.rst (hunk 1 above)
Comment (phw) [reply config/options_before_tagging.rst]: Fixed. I also changed the sentence a bit to make it a bit clearer.

**Review by rdswift — APPROVED** [review]
Looks good.  Thanks.

**Pull request merged**